src/Core/DataAnnotations/RequiredLocalizedAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.DataAnnotations
+{
+	public class RequiredLocalizedAttribute : System.ComponentModel.DataAnnotations.RequiredAttribute
+	{
+
+		private string errorMessageString;
+
+		public RequiredLocalizedAttribute(string errorMessageString)
+		{
+			this.errorMessageString = errorMessageString;
+		}
+
+		public override string FormatErrorMessage(string name)
+		{
+			//return Localizer
+			throw new NotImplementedException();
+		}
+
+	}
+}
```

Comment:
This shouldn't be here, it was testing code that I forgot to remove before creating this request.  I shall do so in the next checkin.